electrum/ecc.py

```diff
@@ -374,6 +374,11 @@ def sign(self, data: bytes, sigencode=None, sigdecode=None) -> bytes:
             sigdecode = get_r_and_s_from_sig_string
         private_key = _MySigningKey.from_secret_exponent(self.secret_scalar, curve=SECP256k1)
         sig = private_key.sign_digest_deterministic(data, hashfunc=hashlib.sha256, sigencode=sigencode)
+        counter = 0
+        while ord(sig[0]) > 127: # grind for low R value https://github.com/bitcoin/bitcoin/pull/13666
+            counter += 1
+            extra_entropy = bfh("%064x" % counter)[::-1]
```

Comment:
wouldn't `int.to_bytes(counter, 32, 'little')` be equivalent, clearer and faster?